storage/pebble/registers.go

```diff
@@ -73,23 +108,35 @@ func (s *Registers) Store(
 			return fmt.Errorf("failed to set key: %w", err)
 		}
 	}
-
-	err := batch.Commit(pebble.Sync)
+	// increment height and commit
+	err := batch.Set(LatestHeightKey(), EncodedUint64(height), nil)
+	if err != nil {
+		return fmt.Errorf("failed to update latest height %d", height)
+	}
+	err = batch.Commit(pebble.Sync)
 	if err != nil {
 		return fmt.Errorf("failed to commit batch: %w", err)
 	}
+	s.latestHeight = height
 
 	return nil
 }
 
-// TODO: Finish implementation after deciding prefixes and/or badger use
-
 // LatestHeight Gets the latest height of complete registers available
 func (s *Registers) LatestHeight() (uint64, error) {
-	panic("not implemented")
+	return s.heightLookup(LatestHeightKey())
 }
 
 // FirstHeight first indexed height found in the store, typically root block for the spork
 func (s *Registers) FirstHeight() (uint64, error) {
-	panic("not implemented")
+	return s.heightLookup(FirstHeightKey())
```

Comment:
Given this is exported and likely will be used by external callers (like the indexer), any reason not to use the cached version?

Reply:
I anticipated this being used concurrently and since the db value updates are atomic, will switch to the mutex implementation.